drafts/use-cases/1.1.security-considerations.md

```diff
@@ -10,23 +10,30 @@ There are several possibilities here. One of them would be to use a *@graph* not
 
 ```json
 {
+    "@context": "/api/context.jsonld",
     "@graph": [
         {
             "@id": "/api",
-            "operation": [
-                {
-                    "@type": "hydra:Operation",
-                    "title": "List events",
-                    "method": "GET",
-                    "returns": "hydra:Collection"
-                },
-                {
-                    "@type": "hydra:Operation",
-                    "title": "Create new event",
-                    "method": "POST",
-                    "expects": "schema:Event"
-                }
-            ]
+            "@type": "hydra:EntryPoint",
+            "#events": {
```

Comment:
I defined `examplevocab:events` in JSON-LD context but in client pseudo code I used full IRI since we shouldn't assume the client uses the same prefix map as JSON-LD context used by the API. I hope this gives us enough to merge this PR and in future PRs consider available options for how to get rid of that `examplevocab:events` all together and only rely on `schema:Event` when client needs to discover the collection of all events in the dataset.